From addaa8b1e368dffe0e73f7118ca7e964ac5a402c Mon Sep 17 00:00:00 2001
From: Christian Hergert <chergert@redhat.com>
Date: Wed, 20 Apr 2022 16:24:29 -0700
Subject: [PATCH 3/4] plugins/flatpak: remove libsoup dep from daemon
X-Developer-Signature: v=1; a=openpgp-sha256; l=718; i=xtexchooser@duck.com;
 h=from:subject; bh=LALejI9GhyJ1ZSTbmDIcrFQHGw4OEI8n4cNf1xoJiXU=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDBlWTk+2RrYues/nd3JRRgJf36yVP36f2H07/d3hxZNOu
 VyZ9bwgrKOUhUGMi0FWTJGlyLDBm1UnnV90WbkszBxWJpAhDFycAjAR/QiG/0kPGmck326WOq8U
 bf/csOvqvXPOm117Y1jCubZ+ud193YyRYdbJaMsKFpEP7MVes4qdBWre7TAJ3+ZtaXSAJ/HqRot
 wZgA=
X-Developer-Key: i=xtexchooser@duck.com; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

We don't need our libsoup in this process, flatpak already has one which
would collide with our libsoup-3.0 anyway.

(cherry picked from commit cbef22b9be2886bf3f637b24ffdfff9922143b0c)
---
 src/plugins/flatpak/daemon/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/plugins/flatpak/daemon/meson.build b/src/plugins/flatpak/daemon/meson.build
index 34acf2b9d69e..992cd85ef114 100644
--- a/src/plugins/flatpak/daemon/meson.build
+++ b/src/plugins/flatpak/daemon/meson.build
@@ -4,7 +4,6 @@ gnome_builder_flatpak_deps = [
   libgiounix_dep,
   libflatpak_dep,
   libostree_dep,
-  libsoup_dep,
 ]
 
 ipc_flatpak_service_src = gnome.gdbus_codegen('ipc-flatpak-service',
-- 
2.49.0

